Helm: Default value for priorityClassName is a map not string

**Describe the bug**:
Setting the `priorityClassName` in the helm values, results in the following error if the Chart is used as dependency.

```
coalesce.go:298: warning: cannot overwrite table with non table for priorityClassName (map[])
```

**Expected behaviour**:
Setting the `priorityClassName` to a string should work even with more recent Helm versions.

**Steps to reproduce the bug**:
Use the following value.yaml

```
priorityClassName: "existing-priority-class"
```

**Additional context**:

This seems to be a new behavior introduced with helm/helm#12469 and present since Helm v3.17 release.

**Environment details**:
- logging-operator version (e.g. 2.1.1): 5.3.0
- Install method (e.g. helm or static manifests): helm
- Helm version: v3.17+

/kind bug

--- a/charts/logging-operator/README.md
+++ b/charts/logging-operator/README.md
@@ -64,7 +64,7 @@ Use `createCustomResource=false` with Helm v3 to avoid trying to create CRDs fro
 | monitoring.serviceMonitor.relabelings | list | `[]` |  |
 | podSecurityContext | object | `{}` | Pod SecurityContext for Logging operator. [More info](https://kubernetes.io/docs/concepts/policy/security-context/) # SecurityContext holds pod-level security attributes and common container settings. # This defaults to non root user with uid 1000 and gid 2000.	*v1.PodSecurityContext	false # ref: https://kubernetes.io/docs/tasks/configure-pod-container/security-context/ |
 | securityContext | object | `{}` | Container SecurityContext for Logging operator. [More info](https://kubernetes.io/docs/concepts/policy/security-context/) |
-| priorityClassName | object | `{}` | Operator priorityClassName. |
+| priorityClassName | string | `nil` | Operator priorityClassName. |
 | serviceAccount.annotations | object | `{}` | Define annotations for logging-operator ServiceAccount. |
 | resources | object | `{}` | CPU/Memory resource requests/limits |
 | nodeSelector | object | `{}` |  |

--- a/charts/logging-operator/values.yaml
+++ b/charts/logging-operator/values.yaml
@@ -105,7 +105,7 @@ securityContext: {}
   #   drop: ["ALL"]
 
 # -- Operator priorityClassName.
-priorityClassName: {}
+priorityClassName: ~
 
 serviceAccount:
   # -- Define annotations for logging-operator ServiceAccount.

